types/rdfjs__environment/FormatsFactory.d.ts

```diff
@@ -1,4 +1,4 @@
-import Formats from './lib/Formats';
+import Formats from './lib/Formats.js';
```

Comment:
With module resolution `node16` or `nodeNext` this would cause failures to resolve imports